spec-0000/index.md

```diff
@@ -33,12 +33,23 @@ Specifically, we recommend that:
 1. Support for Python versions be dropped **3 years** after their initial release.
 2. Support for core package dependencies be dropped **2 years** after their initial release.
 
+{{< admonition note >}}
+Core packages may or may not decide to provide bug fix releases during the full 2 year period after release.
+Therefore, projects may occasionally want to drop support for core package dependencies earlier than recommended by this SPEC.
+For instance, if a newer minimum version of a core package is needed by a project due to a critical bug fix,
+which is not backported to older versions.
+{{< /admonition >}}
+
 ### Core Project Endorsement
 
 <!--
 Briefly discuss what it means for a core project to endorse this SPEC.
 -->
 
+Core project endorsing this SPEC means that those projects encourage all projects across the Scientific Python ecosystem
+to limit how long they support older Python versions and older core packages versions.
```

Comment:
This SPEC only has a table for the core projects, but the driving philosophy is true for all dependencies I believe.

```suggestion
to limit how long they support older Python versions and older dependency versions.
```